### ISSUE TITLE
Bugs in BRouter 1.6.3

### ISSUE THREAD

**Issue by Thunichgud** [abrensch/brouter#399]
- LineageOS 17.1 (Android 10)
- OSMand 4.1.0 NB
- BRouter 1.6.3 build 43 for the API 19

Because of the restrictions of the Google Play version, I use the API version 19 of the BRouter 1.6.3. There are the following bugs:
1. After app start, the BRouter finds and offers the NoGo points from my OSMand favorites. But during route calculation it also uses the areas of the selected NoGo points. It calculates as there are no NoGo points selected. 
2. At the first route calculations after installation of BRouter, the app don't saves the calculated route because it don't can find the directory "router" in the directory of "btools.routingapp". After manual creation of the directory "router", the saving works. I think in the source code of BRouter is a "b" missing in front of the word "router" and the saving directory should be named "brouter", which is created during installation.
3. I created the file "brouter.redirect" in the directory "tracks" of my OSMand directory. In this file I wrote in a single line the path to the calculated "brouter0.gpx" file as described in the "kitkat_survival_readme.txt" from BRouter. This should open the way for OSMand to read the calculated file. But it don't works. OSMand don't finds the calculated file. BRouter finds the "brouter.redirect" file what can be tested by writing a wrong path into this file. In this case BRouter opens an error message. 
Manual copying of the calculated file to the "tracks" directory of OSMand is very very circumstantial. This could not be a feature of BRouter.

**Comment by afischerdev** [comment]
Could you please try out this changes with the current zip file (see actions). 

Fpr the point 1 I have no Osmand installation. And when I install it I found the in /Android/data (not accessable). So CoordinateReader for Osmand not working for me.

**Comment by afischerdev** [comment]
I've tried your description like this:
I have a favourites.gpx in my import folder and two nogo_...gpx in the track folder.
After start I select BRouter App and a profile. Then I got a list with my nogo files 'Check NoGo Selection'.
I unchecked the files and go on.
Selecting two points and calculating a route brings a track that doesn't pass the nogo lines.
When I have the lines checked in 'Check NoGo Selection' dialog it routes over the nogo lines.
I ignore the listed lines in 'Select from' dialog.

But I used the internal coordreader. Only difference with Osmand reader is: internal reads 'trkpt' and 'rtept' and has other folders.
